.github/workflows/NextMinor.yaml

```diff
@@ -125,6 +127,24 @@ jobs:
           settingsJson: ${{ env.Settings }}
           SecretsJson: ${{ env.RepoSecrets }}
 
+      - name: Upload thisbuild artifacts - apps
```

Comment:
```suggestion
      - name: Upload this build artifacts - apps
```

Reply:
Is that some kind of convention I don't know: _thisbuild_?

Reply:
Yeah, this comes from AL-Go. It was updated in [this PR](https://github.com/microsoft/AL-Go/pull/297/files#diff-01f4ef19a96cdb7663a0199b5880386ffc66ae75cf8696fbc655bcc2eb5cc316).

Reply:
The thisbuild artifacts are artifacts from this build, which only is used for this build to carry over to the next project (when running multiprojects with useProjectDependencies = true) - a fix for this issue: https://github.com/microsoft/AL-Go/issues/292